debugger/static/memory.js

```diff
@@ -0,0 +1,62 @@
+class Memory extends HTMLDivElement {
+    static tagName() {
+        return 'memory-div';
+    }
+
+    constructor() {
+        super();
+
+        this.id = "memory";
+
+        this.ram = document.createElement("pre");
```

Comment:
nit: should we add two headers to indicate that the first part is the RAM and the second one the VRAM ?

Reply:
Yes right. do you prefer both tabs side by side or one below the other like it is now ?